src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -376,6 +377,53 @@ public void writeMethodInvocationExpr_chained() {
         writerVisitor.write(), "libraryClient.streamBooksCallable().doAnotherThing().call()");
   }
 
+  @Test
+  public void writeAnonymousClassExpr_basic() {
```

Comment:
Can we also add a test in ImportWriterVisitor?

Reply:
sure, done.